src/deriver/consumer.py

```
@@ -45,36 +44,41 @@ async def process_item(task_type: str, payload: dict[str, Any]) -> None:
+        if settings.LANGFUSE_PUBLIC_KEY:
+            langfuse_context.update_current_trace(  # type: ignore
+                metadata={
+                    "critical_analysis_model": settings.DERIVER.MODEL,
+                }
+            )
+        try:
+            validated = SummaryPayload(**payload)
+        except ValidationError as e:
+            logger.error(
+                "Invalid summary payload received: %s. Payload: %s", str(e), payload
+            )
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)